### PR TITLE
Remove installer specific code

### PR DESCRIPTION
**Pull request by fjvallarino**
@tejohnso @ahmedalsudani please review

### CODE REVIEW

**Comment by tejohnso** [comment]
:+1:

File: proxy.js [hunk 1]
```diff
@@ -14,6 +13,9 @@ function reset() {
 }
 
 module.exports = {
+  setSaveDir(saveDir) {
+    pacScriptPath = path.join(saveDir, "proxy-pac.js");
+  },
```
Comment (ahmedalsudani) [inline proxy.js]: @fjvallarino no need to change in this PR, but in the future, I think we should try to move away from storing state in components and instead provide arguments.

Comment (fjvallarino) [inline proxy.js]: @ahmedalsudani agreed, although the intention here is not saving state, but being able to get information that is no longer accessible (saveDir contains the installerDir, which is now part of the launcher and not of common).

**Comment by ahmedalsudani** [comment]
:+1:

**Pull request merged**